docs/getting-started.md

```diff
@@ -4,16 +4,20 @@
 In order to contribute to foreman-kubevirt integration, there is a need to setup a development environment.
 The development environment should contain two VMs running on the development machine (i.e. laptop).
 
-### Development machine configuration
+### On your PC (Development machine configuration)
 The virtual machines are scheduled on the development machine.
 Foreman development environment is located on the development machine, created by following [Foreman's guide](https://theforeman.org/contribute.html) .
 
-Two libvirt networks are defined:
+Two libvirt networks should be defined:
 * default - provides external connectivity
 * foreman - inner network that connects the VMs (foreman <--> kubevirt <--> foreman-proxy). The network will serve for booting KubeVirt's VM from a PXE server.
   * Address on development machine: 192.168.111.1
-  * The domain *example.tst*
+  * The domain *kubevirt.com*
```

Comment:
this domain actually exists and might lead to unexpected issues. perhaps kubevirt.tst ?

Reply:
it worked for me, but if you want I can change it.

Reply:
let's change it through the guide

Reply:
done